instance/models/instance.py

```diff
@@ -52,14 +52,25 @@ class InstanceReference(TimeStampedModel):
     """
     InstanceReference: Holds common fields and provides a list of all Instances
 
-    Has name, created, and modified fields for each Instance.
+    Has name, created, instance_type_id, and modified fields for each Instance.
 
     Instance is an abstract class, so having this common InstanceReference class gives us a
     fully generic way to iterate through all instances and allow instances to be implemented
     using a variety of different python classes and database tables.
     """
+    INSTITUTIONAL = "0"
```

Comment:
I'm using properties for the types so that I can reference them when adding the default instance purpose with something like Instance(name='name', instance_purpose=InstanceReference.SANDBOX)